cmd/root.go

```diff
@@ -172,7 +173,13 @@ func runSignalWrapper(cmd *Command) error {
 	startCh := make(chan *proxy.Client)
 	go func() {
 		defer close(startCh)
-		p, err := proxy.NewClient(ctx, cmd.Command, cmd.conf)
+		d, err := cloudsqlconn.NewDialer(ctx)
+		if err != nil {
+			defer d.Close()
```

Comment:
Should we close the dialer if err != nil? Doesn't that mean that dialer = nil?

Reply:
I don't know what I was thinking. I'll remove this.